rust/plugin-lib/src/lib.rs

```diff
@@ -109,6 +111,11 @@ pub trait Cache {
 pub trait Plugin {
     type Cache: Cache;
 
+    /// Called when the Plugin is initialized. The plugin receives CoreProxy
+    /// object that is a wrapper around the RPC Peer and can be used to call
+    /// related methods on the Core in a type-safe manner.
+    fn initialize(&mut self, _core: CoreProxy) {}
```

Comment:
We prefer `core` (no underline prefix) and `#[allow(unused)]`, because it makes the docs cleaner.